### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by liamdewitt**

### CODE REVIEW

**Review by digitalRM — APPROVED** [review]

File: app/page.js
Comment (Connor-Skudlarek) [inline app/page.js:1]: This is not intended functionality I think.

The homepage is now the CartPage. Should probably revert this change.

Then add a cartpage folder, and move the CartPage into a page.js in that folder for Next's routing system.

**Review by Connor-Skudlarek — COMMENTED** [review]
Other than the page.js change at the root, I think everything is ok.
